From aa61c7e2818d8514a9c5ad1f216aea57bcbf0f91 Mon Sep 17 00:00:00 2001
From: Peter Steenbergen <info@j1nx.nl>
Date: Wed, 9 Jan 2019 13:19:13 +0100
Subject: [PATCH 1/1] Remove python2 dependency from python-typing Mycroft MSK
 and MSN require it although Python 2.x should not need it.

---
 package/python-typing/Config.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/package/python-typing/Config.in b/package/python-typing/Config.in
index 81b7d71b4b..3d7376f28b 100644
--- a/package/python-typing/Config.in
+++ b/package/python-typing/Config.in
@@ -1,6 +1,5 @@
 config BR2_PACKAGE_PYTHON_TYPING
 	bool "python-typing"
-	depends on BR2_PACKAGE_PYTHON
 	help
 	  Type Hints for Python. This is a backport of the standard
 	  library typing module to Python versions older than 3.5.
-- 
2.11.0

